bats/fb-katello-content.bats

```diff
@@ -71,6 +71,29 @@ setup() {
     --product="${PRODUCT}" --name="${CONTAINER_REPOSITORY}"
 }
 
+@test "create an ostree repository" {
+  if ! tIsRHEL 8 && ! tIsFedora 8; then
```

Comment:
Fedora 8?

Reply:
That makes no sense. I don't know what the thought there was.